src/components/Icon/icons/index.ts

```diff
@@ -1,26 +1,45 @@
 /*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
 
 // Let's try to keep these alphabetized
+import arrowDown from './ArrowDown';
+import arrowLeft from './ArrowLeft';
+import arrowRight from './ArrowRight';
+import arrowUp from './ArrowUp';
 import attach from './Attach';
+import carrotDown from './CarrotDown';
+import carrotLeft from './CarrotLeft';
+import carrotRight from './CarrotRight';
+import carrotUp from './CarrotUp';
 import check from './Check';
 import cross from './Cross';
 import down from './Down';
+import dynamicGroup from './DynamicGroup';
 import edit from './Edit';
 import external from './External';
 import group from './Group';
 import info from './Info';
 import leave from './Leave';
+import like from './Like';
 import lock from './Lock';
 import mail from './Mail';
 import minus from './Minus';
+import more from './More';
 import no from './No';
 import phone from './Phone';
 import plus from './Plus';
 import skype from './Skype';
 import time from './Time';
 
 export default {
+  arrowDown,
+  arrowLeft,
+  arrowRight,
+  arrowUp,
   attach,
+  carrotDown,
+  carrotLeft,
+  carrotRight,
+  carrotUp,
```

Comment:
Why do we call these *carrot*? They surely are *carets*. 🥕🥕🥕🥕

Reply:
haha.

Reply:
Adrian is changing the Sketch file names, and I'm updating this branch now. Thanks for catching this, haha.

Reply:
(and he's going to give us a new minus svg but later)